Updated awssecuritylake (#1488)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md

This makes AWS log setup relatively easy. In addition, the data available from various AWS services is in the OCSF format, making it simple to parse and correlate across different sources.

## Setup Relevant IAM Roles on Amazon Security Lake

Before setting up your Amazon Security Lake account, you need to create the following IAM roles in AWS:

### Role 1 (AmazonSecurityLakeMetaStoreManager)

> Create the `AmazonSecurityLakeMetaStoreManager` role in AWS Identity and Access Management (IAM). The role must carry this name and is necessary for Security Lake to support extract, transform, and load (ETL) jobs on raw log and event data that it receives from sources. Without creating and assuming this role, you cannot create your data lake or query data from Security Lake. One role can be used across Regions—there's no need to create a separate role for different Regions.

Author: "that" is used here as relative clause.

The role name must be in the format: `AmazonSecurityLakeRoleForEventBridge-[AWS Account ID]`. For example, AmazonSecurityLakeRoleForEventBridge-03039853141.
3. After creating this IAM role, the Role ARN will be generated automatically.
4. Copy and secure the Role ARN of the role as you'll need it to create the subscriber in later steps. It must be in this format something like `arn:aws:iam::account-is:role/role-name`. <br/><img src={useBaseUrl('img/send-data/role-arn-name.png')} alt="role-arn-name.png" width="1200"/>

We assume that by following the above steps, you have successfully created the required IAM roles and generated a Role ARN. In the next step, you'll grant Sumo Logic access to Amazon Security Lake to use the services.

## Configure Amazon Security Lake Setup

### Step 1. Grant Sumo logic access to Amazon Security Lake console

To grant Sumo Logic access to the Amazon Security Lake source, you need to provide the Sumo Account ID and External ID.

> 1. **Account ID**. Remember to copy and secure your `Sumo Logic ID`: `************`. You would require this ID in the further steps.

Reviewer: ```suggestion
1. **Account ID**. Remember to copy and secure your `Sumo Logic ID`: `************`. You will require this ID in further steps.
```

> 2. **External ID**. The External ID is formed from your Sumo Logic region identifier. It is a unique identifier that is used to authenticate the connection between your Sumo Logic account and your Amazon Security Lake source. The format of your Sumo Logic account identifier is: `SumoDeployment`: `SumoAccountId` where:

Author: @nmohantyctr Thank you for the suggestion. However, the sentence is correct here, and there is no need for further edits. External ID is formed which means it is obtained.

   * `SumoDeployment` is your Sumo Logic deployment that has to be entered in lowercase such as `au`, `ca`, `de`, `eu`, `fed`, `in`, `jp`, `us1`, or `us2`. To find your deployment, see [Sumo Logic API Authentication, Endpoints, and Security](/docs/api/getting-started.md).
   * `SumoAccountId` is the Organization ID shown on your Account Overview page in the Sumo Logic UI. You can access it by navigating to **Administration** > **Account**  > **Account Overview**.

> After the authentication is done successfully, you'll create the subscriber.

Reviewer: ```suggestion
After the authentication is successful, you'll create the subscriber.
```

### Step 3. Create Amazon Security Lake Source in Sumo Logic

When you create an Amazon Security Lake source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use from the existing ones or create a new Hosted Collector, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
**To create an Amazon Security Lake Source:**
1. In the Sumo Logic environment, go to **Manage Data** > **Collection** > **Collection**.
1. On the **Collectors page** click **Add Source** next to a Hosted Collector.
1. Select an existing Hosted Collector or the one you have just created.
1. Select **Amazon Security Lake**. <br/><img src={useBaseUrl('img/send-data/aws-security-lake.png')} alt="aws-security-lake.png" width="900"/>
1. Enter a name for the new **Source**. The **Description** is optional.
1. In the **Source Category**, enter any string to tag the output collected from this distinct source. (Category metadata is stored in a searchable field called **_sourceCategory**).
7. In **Fields**. Click the **+Add Field** link to add custom log metadata fields.

> 8. Enter the required fields that you want to associate, each field needs a name (key) and value.

Reviewer: ```suggestion
8. Enter the required fields you want to associate, each field needs a name (key) and value.
```

1. Click **Save**. A pop-up will appear with the subscription endpoint. <br/> <img src={useBaseUrl('img/send-data/subscription-endpoint.png')} alt="subscription-endpoint.png" width="400"/>
  :::note
  Remember to copy and secure the subscription endpoint. If you closed the window and could not copy the URL, don't worry. Click on the **Edit** button under the list of sources and copy the URL.
  :::
1. Optional: You can create any **Processing Rules** that you want for the Source. For more information, see [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule.md).

The next step is to update your Amazon Security Lake subscriber to fill in the subscriber endpoint on the Subscribers page.

### Step 4: Update Amazon Security Lake subscriber

To update the Amazon Security Lake subscriber, follow the steps below:
1. Navigate to the Subscribers page and click **My Subscribers**.
2. Edit the subscriber created in the [Setup Subscriber](#step-2-set-up-subscriber-in-amazon-security-lake-console) section in step 2.

> 3. Go to the **Subscription endpoint** field which you left it blank. Paste the subscription endpoint that you copied from the Amazon Security Lake source setup. See [Step 3.10](#step-3-create-amazon-security-lake-source-in-sumo-logic) section.

Reviewer: ```suggestion
3. Go to the **Subscription endpoint** field, which you left blank. Paste the subscription endpoint you copied from the Amazon Security Lake source setup. See [Step 3.10](#step-3-create-amazon-security-lake-source-in-sumo-logic) section.
```
